_posts/research/2020-04-09-cooperative_multirobot_systems.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Cooperative Multi-Robot Systems
+description: Reinforcement Learning for Cooperative Control
+author: Joewie J. Koh
+permalink: research/cooperative_multirobot_systems.html
+image:
+    feature: research/multirobot/cooperative_manipulation.png
+    size: 70%
+excerpt_separator: <!-- More -->
+---
+
+Multi-robot systems offer vastly increased competencies for robotic applications. Multiple robotic agents can, through cooperative control, accomplish common goals beyond the capabilities of single-robot systems. In particular, a cooperative multi-robot system has capabilities that are more than the sum of its parts, e.g., robots can cooperate to manipulate or transport objects that are too large or heavy for a single robot to manage.
```

Comment:
" Multi-robot systems offer vastly increased competencies for robotic applications. " Lame, doesn't mean anything.

Reply:
"  Multiple robotic agents can, through cooperative control, accomplish common goals beyond the capabilities of single-robot systems. " not as effective as it could